src/Raven.Embedded/RavenEmbedded.cs

```diff
@@ -0,0 +1,207 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Diagnostics;
+using System.Text;
+using Raven.Client.Documents;
+using Raven.Client.Exceptions;
+using Raven.Client.Extensions;
+using Raven.Client.ServerWide;
+using Raven.Client.ServerWide.Operations;
+using Sparrow.Logging;
+
+
+namespace Raven.Embedded
+{
+    public class RavenEmbedded
```

Comment:
Somehow I do not like this name... `EmbeddedServer` ? Thoughts?

Reply:
I am fine with RavenEmbedded